task11.py

```diff
@@ -0,0 +1,39 @@
+"""
+Task11.
+
+# #############################################################################
+# write generator function that has as input some range value and chunk_size
+# produces output as lists with len == chunk size
+# Example:
+# Call:  chunk_generator(range(11), chunk_size=3) ->
+# Output:  [0, 1, 2]
+#          [3, 4, 5]
+#          [6, 7, 8]
+#          [9, 10]
+# #############################################################################
+"""
+
+
+def chunk_generator(input_range, chunk_size):
+    """
+    Write Generator function.
+
+    Args:
+        input_range:range(n)
+        chunk_size:int
+    Yields:
+        lists
+    """
+    nl = len(input_range)
+    matrix = [
+        [jl for jl in range(il, il + 3) if jl < nl]
+        for il in range(0, nl, chunk_size)
+    ]
+    for it in matrix:
+        yield it
```

Comment:
```suggestion
for il in range(0, nl, chunk_size):
    yield [jl for jl in range(il, il + chunk_size) if jl < nl]
```
No sense to write matrix and then output values from it. You able to do yield from loop directly